src/borg/archiver.py

```diff
@@ -778,7 +778,7 @@ def peek_and_store_hardlink_masters(item, matched):
         filter = self.build_filter(matcher, peek_and_store_hardlink_masters, strip_components)
         if progress:
             pi = ProgressIndicatorPercent(msg='%5.1f%% Extracting: %s', step=0.1, msgid='extract')
-            pi.output('Calculating size')
+            pi.output('Calculating total archive size for the progress indicator (might take long if it\'s a large one)')
```

Comment:
`... (might take long for large archives)`

avoids the ugly backslash and also clearer.